recipes/effectivet3/effectivet3.py

```diff
@@ -95,7 +95,7 @@ def main():
 
     java_args = [java] + mem_opts + prop_opts + [jar_arg] + [jar_path] + pass_args
 
-    sys.exit(subprocess.call(java_args))
+    sys.exit(subprocess.check_call(java_args))
```

Comment:
Why this change? Won't ``subprocess.call`` just return a non-zero on error, and that 
 return code whatever it is directly passed to the caller of the script via ``sys.exit``

https://docs.python.org/3/library/subprocess.html#subprocess.call

Reply:
Hmm, could be, in which case java isn't returning a non-zero exit on errors...which is very problematic.

Reply:
Ah, I see why these are failing, the tests need to know where these other jar files are.

Reply:
Should the wrapper script handle the `-m something.jar` and convert the jar file path?

Reply:
Changing the path in `-m` won't actually do anything, since the TTSS_GUI jar file will prepend ./module to it. The only way I can see to get this to work is to either actually run it from within the directory containing the TTSS_GUI jar file or to symlink the module subdirectory into the current working directory. Both of these are kind of crappy options. Symlinking modules and then removing the symlink in the script is probably the simplest work-around, but it's prone to breaking if you have multiple runs in parallel in the same directory. Changing into the directory with TTSS_GUI.jar and then having the wrapper script translate the input and output file paths (or configuration file path) is probably the better option. Ideally the package authors would allow providing a path to the modules directory, but I assume this is no longer maintained since there hasn't been a new version released in years.